sound/soc/sof/ipc4-ops.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
+ */
+
+#ifndef __SOUND_SOC_SOF_IPC4_OPS_H
+#define __SOUND_SOC_SOF_IPC4_OPS_H
+
+#include "sof-priv.h"
+
+extern const struct ipc_tplg_ops ipc4_tplg_ops;
+/* tplg_ipc4_control_ops is not implemented yet, useing tplg_ipc3_control_ops for now */
```

Comment:
typo useing

Reply:
but why not omit them instead?

Reply:
> but why not omit them instead?

Because control ops is mandatory https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/ipc.c#L1014

Reply:
Oh I see.ok then